libs/core/src/lib/grid-list/components/grid-list/grid-list.component.scss

```diff
@@ -1,4 +1,4 @@
 @import '~fundamental-styles/dist/radio';
 @import '~fundamental-styles/dist/checkbox';
 @import '~fundamental-styles/dist/layout-grid';
-@import '~fundamental-styles/dist/grid-list';
+@import '~fundamental-styles/dist/grid-list';
```

Comment:
there are no more changes, is it intended?

Reply:
trying to make the pr not fail.. it kept failing to package.json